docs/faq.rst

```diff
@@ -334,6 +334,145 @@ After successfull login the domain part will be striped and the rest used as use
 .. _`591`: https://github.com/Mailu/Mailu/issues/591
 .. _`575`: https://github.com/Mailu/Mailu/issues/575
 
+Spam filtering
+--------------
+
+How does spam filtering work in Mailu?
+``````````````````````````````````````
+
+Mailu uses Rspamd for spam filtering. Rspamd is a Fast, free and open-source spam filtering system.
+
+Rspamd rejects non-compliant email messages and email messages that contain virusses. In Mailu Rspamd uses a scoring scale from 0 to 15:
+
+* Email messages with a score of 15 or higher will be rejected.
+
+* Email messages with a score of 6 or higher will receive a header with the spam score.
+
+* Email messages with a score of 4 or higher will be greylisted.
+
+* Email messages with a score of 3 or lower will be accepted.
+
+1. When an email message is received or send by Postfix, it is scanned by Rspamd. If the email message receives a spam score between 6 and 15, a header is added to the email message with the spam score. This is an example of an email header for a spam score of 14:
+
+  ``X-Spam-Level: **************``
+
+2. Dovecot is then responsible for classifying the email message to the Junk folder based on user preferences. It works as following:
+
+   * In the administration web interface, under settings under Antispam 'Enable spam filter' must be ticked. If this option is disabled, then all email messages will automatically go to the inbox folder. Except for email messages with a score of 15 or higher, as these email messages are rejected by rspamd.
+   
+   * In the administration web interface, under settings under Antispam, the user defined spam filter tolerance must be configured. The default value is 80%. The lower the spam filter tolerance, the more false positives (ham classified as spam). The user can change this setting to finetune when an email message is classified as spam.
+
+   * Dovecot extracts the X-Spam-Level email header from the email message and converts the spam score (0 - 15) to a 0 - 100 percent scale. This spam score is compared with the user defined spam filter tolerance. If the spam score is lower than the user defined spam filter tolerance, then the email message is accepted. In logic:
+     
+     If <spam_score in percent> is greater than <spam filter tolerance>, then move the email message to the spam folder and mark the email message as read.
+   
+   For example if the user defined spam filter tolerance is set to 80%(default) and the spam score of an email message is 10:
+      
+   66% (10/15) is less than 80%, so the email is classified as ham. This email message will go to the inbox folder. If the user wants email messages with a score of 10 (66%) to be classified as spam, then the user defined spam filter tolerance can be lowered to 65% in the administration web interface.
+   
+.. image:: assets/screenshots/SpamFiltering.png
+
+The location in the administration web interface where the spam filter and spam filter tolerance can be configured.
```

Comment:
If we keep this as a simple FAQ, maybe that is too much?
If you want to include it, please provide either a link to more documentation which describes how this can be configured, or shortly summarize it here.

Reply:
I'm keeping this because I'm moving everything to a separate page as suggested by  Nebukadneza.